test/mean-module.test.js

```diff
@@ -71,7 +73,11 @@ describe('Module Sub Generators Tests', function () {
       assert.file('modules/foo/client/services/');
       assert.file('modules/foo/client/views/');
       assert.file('modules/foo/tests/client/');
-      assert.file('modules/foo/server/config');
+      assert.file('modules/foo/server/config/');
+      assert.file('modules/foo/server/controllers/');
+      assert.file('modules/foo/server/models/');
+      assert.file('modules/foo/server/policies/');
+      assert.file('modules/foo/server/routes/');
       assert.file('modules/foo/tests/server/');
     });
```

Comment:
It's quite simple - the test was wrong. Now I have updated both the test and the implementation, and folders are created properly.